### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by antocuni**
As the name says.
This also applies the "needs-triage" label to all new issue, so that we can easily filter them.

### CODE REVIEW

**Comment by jaynarayan-vaishnav** [comment]
hey @antocuni ,
if I am guessing correctly then you should also look at issue #255 . I think this is related to your PR and there are  more issue templates required for different categories.

**Comment by antocuni** [comment]
yes, I wasn't aware of #255. I'll write a comment there

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: needs-triage
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (fpliger) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: I'd suggest adding a less negative way of framing the feature request to encourage positive feedback and healthy discussions. Something like "As a user, I'd like to..."

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (fpliger) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: want -> like/expect ?

**Review by fpliger — CHANGES_REQUESTED** [review]
Awesome! This will be super helpful. I'd only think we should address or discuss my comment on the wording to encourage positive feedback.

**Comment by antocuni** [comment]
> Awesome! This will be super helpful. I'd only think we should address or discuss my comment on the wording to encourage positive feedback.

done.
Note that the previous wording is the default github template for feature requests, but I agree with you that encouraging healthy discussions is helpful.

**Comment by fpliger** [comment]
Ehehe.. Github, if you are listening, I'd suggest to improve this! 😆

**Review by fpliger — APPROVED** [review]
Looks great!! TY !

**Pull request merged**